src/drive/web/modules/filelist/File.spec.jsx

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
Why the `use strict` here?

Reply:
For no particular reason. I think I saw it in another file and thought that it is better to use it when it's possible. Do you want me to remove it?